.github/config/evergreen-arm-softfp.json

```diff
@@ -13,8 +13,9 @@
   },
   "platforms": [
     "evergreen-arm-softfp",
+		"evergreen-arm-softfp-sbversion-14",
```

Comment:
Something funny with formatting here ( tabs vs spaces ? )

Reply:
Oh yeah, I fixed a couple of other instances. I must've missed this one.

Whoever added sbversion-14 did it all with tabs.